js/index.js

```diff
@@ -1 +1,62 @@
 // Your code goes here
+
+//Change Fun Bus Title - Mouseover
+
+const funBusTitle = document.querySelector('.logo-heading');
+funBusTitle.addEventListener(
+    "mouseover",
+        event => {
+            event.target.style.color = "red";
+        setTimeout(function() {
+            event.target.style.color = "";
+        }, 500);
+    },
+    false
+    );
+    
+
+// Change background red - Keydown
+
+const backgroundRed = document.querySelector('html');
+backgroundRed.addEventListener(
+    "keydown",
+        event => {
+            event.target.style.background = "red";
+        setTimeout(function() {
+            event.target.style.background = "";
+        }, 500);
+    },
+    false
+    );
+
```

Comment:
these are all really good! I would have liked to see you complete all 10 of them to gain more practice. If you got stuck on the drag, don't hesitate to skip it and move on to the next one, then go back to it. Also, if you hit a wall, reach out!